.golangci.yml

```diff
@@ -13,7 +13,6 @@ issues:
 
 output:
   formats:
-    - format: github-actions
```

Comment:
`WARN [config_reader] The output format `github-actions` is deprecated, please use `colored-line-number`  Been the case that github actions supports `colored-line-number` for a while, just now making the changes